.settings/org.eclipse.jdt.apt.core.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
I think that .settings folder should be placed inside .gitignore and dockerignore, and removed from vcs